e2e/cep47/usage.ts

```diff
@@ -67,7 +67,7 @@ const test = async () => {
 
   const contractPackageHash = await getAccountNamedKeyValue(
     accountInfo,
-    `contract_package_hash`
+    `${CONTRACT_NAME!}_contract_package_hash`
```

Comment:
As discussed, even if this is a backport, I would suggest to remove all those exclamation marks somehow because it acts as a non-null assertion operator so it's a bit strange reading that for a configuration const.